Default Hibernate 2LC configuration fails to load due to out-dated default jgroups configuration location

### Describe the bug

Since https://github.com/infinispan/infinispan/commit/f166d60d69cd7030edb43ac4d8b7030f3e29b4ea the resource location of the default jgroups configuration files changed from "/default-configs/" to "/org/infinispan/configuration/".
However, the default Hiberaten 2LC configuration still references the old location:
https://github.com/infinispan/infinispan/blob/main/hibernate/cache-commons/src/main/resources/org/infinispan/hibernate/cache/commons/builder/infinispan-configs.xml#L8C91-L8C130

Consequently, if the cache manager used by the 2LC is non-local, the 2LC will *always* fail to start:
```
org.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.cache.spi.CacheImplementor] due to: HHH025009: Unable to start region factory
	at org.hibernate@6.6.31.Final//org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:276)
	at org.hibernate@6.6.31.Final//org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:238)
	at org.hibernate@6.6.31.Final//org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:215)
	at org.hibernate@6.6.31.Final//org.hibernate.service.internal.SessionFactoryServiceRegistryImpl.getService(SessionFactoryServiceRegistryImpl.java:114)
	at org.hibernate@6.6.31.Final//org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:245)
	at org.hibernate@6.6.31.Final//org.hibernate.boot.internal.SessionFactoryBuilderImpl.build(SessionFactoryBuilderImpl.java:463)
	at org.hibernate@6.6.31.Final//org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.build(EntityManagerFactoryBuilderImpl.java:1517)
	... 13 more
Caused by: org.hibernate.cache.CacheException: HHH025009: Unable to start region factory
	at org.infinispan.hibernate-cache@16.0.1//org.infinispan.hibernate.cache.v6.InfinispanRegionFactory.start(InfinispanRegionFactory.java:238)
	at org.hibernate@6.6.31.Final//org.hibernate.cache.internal.EnabledCaching.<init>(EnabledCaching.java:85)
	at org.hibernate@6.6.31.Final//org.hibernate.engine.spi.CacheInitiator.initiateService(CacheInitiator.java:31)
	at org.hibernate@6.6.31.Final//org.hibernate.engine.spi.CacheInitiator.initiateService(CacheInitiator.java:23)
	at org.hibernate@6.6.31.Final//org.hibernate.service.internal.SessionFactoryServiceRegistryImpl.initiateService(SessionFactoryServiceRegistryImpl.java:76)
	at org.hibernate@6.6.31.Final//org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:263)
	... 19 more
Caused by: org.infinispan.commons.CacheConfigurationException: ISPN000557: Unable to configure JGroups Stack 'hibernate-jgroups'
	at org.infinispan.core@16.0.1//org.infinispan.configuration.parsing.Parser.addJGroupsStackFile(Parser.java:504)
	at org.infinispan.core@16.0.1//org.infinispan.configuration.parsing.Parser.parseStackFile(Parser.java:611)
	at org.infinispan.core@16.0.1//org.infinispan.configuration.parsing.Parser.parseJGroups(Parser.java:463)
	at org.infinispan.core@16.0.1//org.infinispan.configuration.parsing.Parser.readElement(Parser.java:94)
	at org.infinispan.core@16.0.1//org.infinispan.configuration.parsing.ParserRegistry.parseElement(ParserRegistry.java:213)
	at org.infinispan.core@16.0.1//org.infinispan.configuration.parsing.ParserRegistry.parse(ParserRegistry.java:196)
	at org.infinispan.core@16.0.1//org.infinispan.configuration.parsing.ParserRegistry.parse(ParserRegistry.java:182)
	at org.infinispan.core@16.0.1//org.infinispan.configuration.parsing.ParserRegistry.parse(ParserRegistry.java:165)
	at org.infinispan.hibernate-cache@16.0.1//org.infinispan.hibernate.cache.commons.DefaultCacheManagerProvider.parse(DefaultCacheManagerProvider.java:94)
	at org.infinispan.hibernate-cache@16.0.1//org.infinispan.hibernate.cache.commons.DefaultCacheManagerProvider.lambda$loadConfiguration$0(DefaultCacheManagerProvider.java:66)
	at org.hibernate@6.6.31.Final//org.hibernate.boot.registry.classloading.internal.ClassLoaderServiceImpl.workWithClassLoader(ClassLoaderServiceImpl.java:272)
	at org.infinispan.hibernate-cache@16.0.1//org.infinispan.hibernate.cache.commons.DefaultCacheManagerProvider.loadConfiguration(DefaultCacheManagerProvider.java:71)
	at org.infinispan.hibernate-cache@16.0.1//org.infinispan.hibernate.cache.v6.InfinispanRegionFactory.defineDataTypeCacheConfigurations(InfinispanRegionFactory.java:339)
	at org.infinispan.hibernate-cache@16.0.1//org.infinispan.hibernate.cache.v6.InfinispanRegionFactory.start(InfinispanRegionFactory.java:234)
	... 24 more
Caused by: java.net.MalformedURLException: no protocol: default-configs/default-jgroups-tcp.xml
	at java.base/java.net.URL.<init>(URL.java:735)
	at java.base/java.net.URL.<init>(URL.java:630)
	at org.infinispan.commons@16.0.1//org.infinispan.commons.configuration.io.URLConfigurationResourceResolver.resolveResource(URLConfigurationResourceResolver.java:19)
	at org.infinispan.core@16.0.1//org.infinispan.configuration.parsing.Parser.addJGroupsStackFile(Parser.java:499)
	... 37 more

```

### Expected behavior

_No response_

### Actual behavior

_No response_

### How to Reproduce?

_No response_

--- a/hibernate/cache-commons/src/main/resources/org/infinispan/hibernate/cache/commons/builder/infinispan-configs.xml
+++ b/hibernate/cache-commons/src/main/resources/org/infinispan/hibernate/cache/commons/builder/infinispan-configs.xml
@@ -5,7 +5,7 @@
         xmlns="urn:infinispan:config:${infinispan.core.schema.version}">
 
    <jgroups>
-      <stack-file name="hibernate-jgroups" path="${hibernate.cache.infinispan.jgroups_cfg:default-configs/default-jgroups-tcp.xml}"/>
+      <stack-file name="hibernate-jgroups" path="${hibernate.cache.infinispan.jgroups_cfg:org/infinispan/configuration/default-jgroups-tcp.xml}"/>
    </jgroups>
 
    <cache-container name="SampleCacheManager" statistics="false" default-cache="the-default-cache" shutdown-hook="DEFAULT">

